utils/man/scoutfs.8

```diff
@@ -57,18 +57,17 @@ a faster block device for the metadata device.
 The path to the block device to be used for ScoutFS file data.  If possible, use
 a larger block device for the data device.
 .TP
-.B "-Q, --quorum-count NUM"
-The number of mounts needed to reach quorum and elect one
-to be the server.  Mounts of the filesystem will hang until a quorum of
-mounts are operational.
-.sp
-Mounts with the
-.B server_addr
-mount option participate in quorum.  The safest quorum number is the
-smallest majority of an odd number of participating mounts.  For
-example,
-two out of three total mounts.  This ensures that there can only be one
-set of mounts that can establish quorum.
+.B "-Q, --quorum-slot NR:ADDR:PORT"
+Each \-Q option configures a quorum slot.  The NR specifies the number
+of the slot to configure, must be between 0 and 14, and only specified
+by one option.  ADDR and PORT are the numerical IPv4 address and port
+which will be used as the UDP endpoint for leader elections and as the
+TCP listening address for server connections.  The number of configured
+slots determines the size of the quorum of member mounts which must be
+present to start the server for the filesystem to operate.  A simple
+majority is typically required, while 1 mount is sufficient if only one
+or two slots are configured.  Until the majority quorum are present, all
+mounts will hang waiting for a server to connect to.
```

Comment:
putting evil user hat on, can these be specified out of order? Can they be non consecutive? Repeating a NR is an error, I think I saw somewhere, yes?

Reply:
yes, yes, correct :).  I'll try to say so specifically in the man page.